src/main/java/dev/xpple/simplewaypoints/commands/WaypointCommand.java

```diff
@@ -33,6 +34,7 @@
 
 public class WaypointCommand {
     private static final SimpleWaypointsAPI API = SimpleWaypointsAPI.getInstance();
+    private static final SimpleCommandExceptionType WAYPOINT_NOT_FOUND_EXCEPTION = new SimpleCommandExceptionType(Component.translatable("commands.sw:waypoint.notFound"));
```

Comment:
The translation key expects an argument, you should make it a `DynamicCommandExceptionType`.